src/Stratis.Bitcoin.Features.Consensus.Tests/Rules/CommonRules/RulesRegistrationOrderTest.cs

```diff
@@ -54,9 +57,10 @@ public void StratisMainPowConsensusRulesRegistrationTest()
         [Fact]
         public void StratisMainPosConsensusRulesRegistrationTest()
         {
-            new FullNodeBuilderConsensusExtension.PosConsensusRulesRegistration().RegisterRules(this.network.Consensus);
+            Network network = new StratisTest();
```

Comment:
This is weird change. The test name specifically includes Stratis main and you're now using Stratis test.?

Reply:
ah yes true, didn't see that.
There is no difference between the two so I'll remove Main from the test names.